requirements.txt

```diff
@@ -24,12 +24,12 @@ dealer
 flask-collect
 flake8
 passlib
-twill
+https://github.com/alexandre/twill/tarball/master
```

Comment:
are you going to submit it to original author, or create a pip module for your twill?